doc/source/data/transforming-datasets.rst

```diff
@@ -130,16 +130,26 @@ Here is an overview of the available batch formats:
     This may incur a conversion cost if the underlying Dataset block is not
     zero-copy convertible from an Arrow table.
 
+  .. literalinclude:: ./doc_code/transforming_datasets.py
+    :language: python
+    :start-after: __writing_default_udfs_tabular_begin__
+    :end-before: __writing_default_udfs_tabular_end__
```

Comment:
Nit: Match the indentation of the bullet block so the code example is properly indented.
```suggestion
    .. literalinclude:: ./doc_code/transforming_datasets.py
      :language: python
      :start-after: __writing_default_udfs_tabular_begin__
      :end-before: __writing_default_udfs_tabular_end__
```
<img width="802" alt="Screen Shot 2022-09-29 at 5 25 01 PM" src="https://user-images.githubusercontent.com/7699919/193152914-d89dbe22-eea0-49cc-a854-b3fc5dff5883.png">